api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java

```diff
@@ -194,26 +195,31 @@ private RdrResearcher toRdrResearcher(DbUser dbUser) {
                     RdrExportEnums.ethnicityToRdrEthnicity(
                         dbDemographicSurvey.getEthnicityEnum()))));
       }
-      if (dbDemographicSurvey.getSexAtBirthEnum() != null) {
-        researcher.setSexAtBirth(
-            dbDemographicSurvey.getSexAtBirthEnum().stream()
-                .map(RdrExportEnums::sexAtBirthToRdrSexAtBirth)
-                .collect(Collectors.toList()));
-      }
+      researcher.setSexAtBirth(
+          Optional.ofNullable(
+                  dbDemographicSurvey.getSexAtBirthEnum().stream()
+                      .map(RdrExportEnums::sexAtBirthToRdrSexAtBirth)
+                      .collect(Collectors.toList()))
+              .orElse(new ArrayList<SexAtBirth>()));
```

Comment:
Looks good. Passing in `.ifPresent(researcher::setSexAtBirth)` is more a matter of taste.